index.html

```diff
@@ -16,123 +23,140 @@
-            <span class="translation">No Pursuit of Personal Advantage</span>
-          6. Không nói dối
-            <span class="translation">No Lying</span>
+          <div class="six-great-principles">
+            <div class="six-great-principles-text">
+              1. Không tranh
+              <span class="translation">No Fighting</span>
+              2. Không tham
+              <span class="translation">No Greed</span>
+              3. Không mong cầu
+              <span class="translation">No Seeking</span>
+              4. Không ích kỷ
+              <span class="translation">No Selfishness</span>
+              5. Không tự lợi
+              <span class="translation">No Pursuit of Personal Advantage</span>
+              6. Không nói dối
+              <span class="translation">No Lying</span>
+            </div>
+          </div>
+          <div class="master">
+            <img
+              src="assets/RevThichHangDatandHisTeacher.jpg"
+              alt="Rev Thich Hang Dat and His Teacher"
+            />
+            HT. Tuyên Hóa<br />và Thầy Thích Hằng Đạt
+          </div>
+          <div class="dalai">
+            <img
+              src="assets/Dalai_Lama_and_Rev_Thich_Hang_Dat.jpg"
+              alt="Dalai Lama and Rev Thich Hang Dat"
+            />
+            Đức Đạt Lai Lạt Ma
           </div>
         </div>
-        <div class="master">
-        <img
-          src="assets/RevThichHangDatandHisTeacher.jpg"
-          alt="Rev Thich Hang Dat and His Teacher"
-        />
-        HT. Tuyên Hóa<br>và Thầy Thích Hằng Đạt
-        </div>
-        <div class="dalai">
-        <img
-          src="assets/Dalai_Lama_and_Rev_Thich_Hang_Dat.jpg"
-          alt="Dalai Lama and Rev Thich Hang Dat"
-          />
-        Đức Đạt Lai Lạt Ma
-        </div>  
       </div>
       <div class="content">
-        <div class="news-title">
-          News
-        </div>
-          <div class="announcements">
-            <div class="announcements-left">
-             <img 
-            src="assets/Quan Am Poster.png"
-            alt="Quam Am Bodhisattva"
-            height="315px"
-            width="430px"
-             />
-            </div>
-            <div class="announcements-right">
-              <img class="announcement-images"
-            src="assets/freeTutor.jpeg"
-            alt="Free Tutoring Service"
-            height="315px"
-            width="430px"
-              />
-            </br>All Ages - Monday-Thursday: 5:30-7:30 PM EST
-            </div>
+        <div class="news-title">News</div>
+        <div class="announcements">
+          <div class="announcements-left">
+            <img
+              class="announcement-images"
+              src="assets/lunar-dragon.jpg"
+              alt="Lunar New Years Animal - Dragon"
+              height="315px"
+              width="430px"
+            />
+            <div>Lunar New Years - Year of the Dragon</div>
+            <div>February 10, 2024</div>
```

Comment:
Thay said it's celebrated the 18th I believe

Reply:
Please review commit ID, [91b92fe](https://github.com/Tu-Vien-Van-Phat-Danh/tu-vien-van-phat-danh.github.io/pull/6/commits/91b92feca30c5d51c0e6c5bbf3b3106a879cec25).

Reply:
LGTM